### DISCUSSION TITLE
Initial State Orbital Element Errors

### DISCUSSION THREAD

**Discussion by hlizsolera** [ARCLab-MIT/STORM-AI-devkit-2025#7] [Announcements]
Hello all,

It has come to our attention that there are some issues with the orbital elements in the initial state file due to a unit error when we fed the original satellite data into our orbital element calculator. We are running the calculations again now and will upload the corrected file as soon as they are finished.

Best regards,
Liz

**Comment by beckja** [comment]
For anyone still using the warmup data, the ENTIRE warmup initial states file (the version currently in Dropbox) has eccentricities greater than .9, altitudes above 400,000 km (1000 times too high), and the semi-major axis is less than the radius of the earth, which is physically impossible for a valid closed orbit. These issues are not present in the phase 1 data.

**Comment by hlizsolera** [comment]
The initial state file in the v2 warmup folder has been updated to reflect the corrected orbital elements that were released in the Phase 1 training data.
